doc/conf.py

```diff
@@ -28,28 +28,48 @@
     "constantdict":
         ("https://matthiasdiener.github.io/constantdict/", None)
 }
+
 autodoc_type_aliases = {
     "Expression": "Expression",
     "ArithmeticExpression": "ArithmeticExpression",
 }
 
-
-import sys
-
-
 nitpick_ignore_regex = [
-    # Avoids this error in pymbolic.typing.
-    # <unknown>:1: WARNING: py:class reference target not found: ExpressionNode [ref.class]  # noqa: E501
-    # Understandable, because typing can't import primitives, which would be needed
-    # to resolve the reference.
-    ["py:class", r"ExpressionNode"],
-    ["py:class", r"_Expression"],
-    ["py:class", r"p\.AlgebraicLeaf"],
-
     # Sphinx started complaining about these in 8.2.1(-ish)
     # -AK, 2025-02-24
     ["py:class", r"TypeAliasForwardRef"],
```

Comment:
This still seems to stick around. At least in the `missing-reference` call, the node that corresponds to this just says `TypeAliasForwardRef` with no information about what it's forwarding to.. so we can't dig that out :\